{"name": "Отображается сообщение об ошибке после запроса без id", "status": "passed", "start": 1733650199566, "stop": 1733650199641, "uuid": "0abc8e08-e030-4b78-8f25-3bee8f0a7bf2", "historyId": "7dd27198fcccd4dcf275c08d7c1560e3", "testCaseId": "7dd27198fcccd4dcf275c08d7c1560e3", "fullName": "tests.test_get_order_using_his_number.TestGetOrder#test_error_message_after_request_without_track_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order_using_his_number"}, {"name": "subSuite", "value": "TestGetOrder"}, {"name": "host", "value": "WIN-3SKQPE0FURM"}, {"name": "thread", "value": "12912-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order_using_his_number"}], "links": [{"type": "link", "url": "https://qa-scooter.praktikum-services.ru", "name": "https://qa-scooter.praktikum-services.ru"}]}